### ISSUE TITLE
Android build will also trigger *-Info.plist change

### ISSUE THREAD

**Issue by pke** [dpa99c/cordova-custom-config#58]
I have this 

```xml
<platform name="ios">
        <config-file overwrite="true" parent="UISupportedInterfaceOrientations" target="*-Info.plist" platform="ios">
            <array>
                <string>UIInterfaceOrientationPortrait</string>
            </array>
        </config-file>
        <config-file overwrite="true" parent="UISupportedInterfaceOrientations~ipad" target="*-Info.plist" platform="ios">
            <array>
                <string>UIInterfaceOrientationPortrait</string>
            </array>
        </config-file>
```

in my `config.xml`. When building an android version thought, this happens:

```
$ cordova build android --release
cordova-custom-config: Applied custom config from config.xml to xxx\platforms\ios\xxx\xxx-Info.plist
```

This doesn't seem to be right.

**Comment by dpa99c** [comment]
The plugin uses an [after_prepare hook](https://github.com/dpa99c/cordova-custom-config/blob/master/plugin.xml#L24) to apply any custom config defined in config.xml to each platform folder under `platforms/`. Regardless of whether you run `cordova build android`, `cordova build ios`, or `cordova build`, the hook script is executed with the same by Cordova. The arguments that Cordova passes to the hook script does not include which platforms it is operating on,

**Comment by pke** [comment]
ok, so that's not really an issue than? Nothing can go wrong and we can just ignore the message?

**Comment by dpa99c** [comment]
yeah, so long as it isn't actually causing a build failure, I think it's OK as it is for now.

**Comment by pke** [comment]
Hmm I see that cordova passes the platforms to the hook in `CORDOVA_PLATFORMS`.
Also see here: https://cordova.apache.org/docs/en/latest/guide/appdev/hooks/#sample-usage

**Comment by dpa99c** [comment]
My bad, so Cordova does inform which platforms are being built. However, as stated above, since this isn't causing any issues other than applying config to platforms not currently being built, it's far from the top of my priority list.

**Comment by pke** [comment]
Created #59 to solve this ;)

**Comment by dpa99c** [comment]
Resolved by #59 - thanks to @pke
